code-challenges.js

```diff
@@ -30,7 +42,12 @@ var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "tem
 var mixedDataArray = [true, 8, "hello", 90, -8, null, 0, 46, 59, 107, "hey!"]
 // Expected output: [-8, 0, 8, 46, 59, 90, 107]
 
-
+//filter through the numbers and then use sort to orger it by
+const numsOnly = (arr) => {
+	let nums = arr.filter(element => typeof element === "number").sort((a,b) => a - b);
+	return nums;
+}
+console.log(numsOnly(mixedDataArray));
```

Comment:
you can return this all in one line if youd like to save yourself some time! 
ex: `return arr.filter(element => typeof element === "n...`